react/features/base/participants/functions.ts

```diff
@@ -267,6 +267,29 @@ export function getVirtualScreenshareParticipantOwnerId(id: string) {
     return id.split('-')[0];
 }
 
+/**
+ * Returns owner participant IDs of the virtual screenshares participant.
+ *
+ * @param {(Function|Object)} stateful - The (whole) redux state, or redux's.
+ * @returns {(string[])}
+ */
+export function getVirtualScreenshareParticipantOwnerIds(stateful: IStateful) {
```

Comment:
Why do we need a new method for this ? Doesn't [getScreenshareParticipantIds](https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/participants/functions.ts#L459) suffice ?

Reply:
@dudumanbogdan ^^

Reply:
Indeed, we are good to go with the proposed existing method, I didn't saw it before. Thank you!! :D